data_process.py

```diff
@@ -39,8 +39,8 @@ def get_traj(self):
                     traj.append([self.raw_data[1][i], self.raw_data[0][i], self.raw_data[-1][i], self.raw_data[-2][i]])
             traj = np.reshape(traj, [-1, 4])
 
-            if self.traj_filter(traj, dup_threshold=self.dup_threshold):
-                self.traj_data.append(traj)
+            #if self.traj_filter(traj, dup_threshold=self.dup_threshold):    don't know whats traj_filter or dup_threshold
+            self.traj_data.append(traj)
```

Comment:
if the file does not exists then why is get_distance_map still used in this code block?